static/src/javascripts/projects/common/modules/article/twitter.js

```diff
@@ -45,9 +42,17 @@ define([
             if (((scrollTop + (viewportHeight * 2.5)) > elOffset.top) && (scrollTop < (elOffset.top + elOffset.height))) {
                 fastdom.write(function () {
                     $(element).removeClass('js-tweet').addClass('twitter-tweet');
+                    // We only want to render tweets once the class has been added
+                    renderTweets();
                 });
             }
         });
+    }
+
+    function renderTweets() {
+        var scriptElement,
+            nativeTweetElements = qwery('blockquote.twitter-tweet'),
```

Comment:
Does it need it? The function is called in fastdom.write, and getting the elements shouldn't cause a reflow, right? Happy to update if something is happening in qwery that I am missing! :)